onnxruntime/python/tools/transformers/models/llama/requirements-70b-model.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
+git+https://github.com/frankdongms/transformers.git@frdong/shard_llama
```

Comment:
We want to merge our changes into HF transformers instead of patch, here is PR: https://github.com/huggingface/transformers/pull/27119

Currently this PR is waiting for review while we want to catch release for this PR so we install transformers from private fork first, after PR in HF transformers got merged we could install from transformers main fork